README.md

```diff
@@ -1,3 +1,138 @@
+
+### Jailbreak Bypassing
+
+Some games refuse to work on macOS because they detect the environment as a jailbroken device. PlayTools hijacks the detection process of popular jailbreak detection tools, so that the game will think it is running on a non-jailbroken device.
+
+PlayTools also allows per-game configuration for more advanced jailbreak bypassing techniques.
+
+### PlayChain
+
+PlayChain is a tool that solves key chain issues by replacing the game's key chain with a custom one. Key chain issues usually prevent the game from logging in.
+
+### Introspection Library
+
+Some games only work under debug environment. By inserting the Introspection Library, PlayTools tricks the game into thinking it is running under debug environment.
+
+# How to Use
+
+PlayTools is shipped with PlayCover, and is installed together with the game. You do not need to install PlayTools manually.
+
+PlayTools provides in-game menus mostly for key mapping setup. Other features are accessible through the PlayCover settings menu.
+
+Localization is handled in [Weblate](https://hosted.weblate.org/projects/playcover/).
+
+# How it Works
+
+PlayTools runs alongside of the game. During IPA installation, PlayTools is inserted into the game's executable. When the game is launched, PlayTools will be launched together.
+
+PlayTools uses swizzle techniques to replace framework methods and system calls with the versions provided by PlayTools. During game play, PlayTools intercepts game input events and translates them to the corresponding touch events and feed them to the game.
+
+# How to Build
+
+PlayTools is built using Xcode.
+
+## Production Build
+In release builds, the building script of PlayCover will automatically fetch the latest version of PlayTools from the official repository and build it. Generally, you do not need to build PlayTools manually.
+
+If you want to build PlayCover with a specific version of PlayTools, mostly for testing purposes, you can change the [Cartfile](https://github.com/PlayCover/PlayCover/blob/develop/Cartfile) of PlayCover to point to the specific version of PlayTools. You can specify which branch/tag of which repository to build from.
+
+## Development Build
+PlayTools can also be built separately. This is useful when you want to modify the source code.
+
+To do so, 
+
+1. Clone the PlayTools repository.
+
+1. Open the PlayTools project in Xcode.
+
+1. Set the development team in the Xcode project settings. Both `PlayTools` and `AKInterface` targets need to be set. You may need to create a development team first.
+
+1. Build PlayTools towards iOS platform. This will create a `PlayTools.framework` in the build directory.
+
+2. Transform the target platform to Mac Catalyst. Run this script in the build directory:
+```bash
+#!/bin/sh
+echo "Converting to maccatalyst"
+vtool \
+	-set-build-version maccatalyst 11.0 14.0 \
+	-replace -output \
+	"PlayTools.framework/PlayTools" \
```

Comment:
Can you give the directory where PlayTools is built to?

Reply:
The directory path is not fixed. It contains directory names such as `PlayTools-cyggpcmfqzdtxnerugmvmfbvzdtw` which looks mostly random. I usually right click on the PlayTools binary in the `Product` group in Xcode and choose "Show in Finder" to find this path.

Reply:
Maybe provide both in the README, but with the path something like `~/Library/some path/PlayTools-UUID/` to represent the random characters.